content/service_dialogs/transform-vm.yml

```diff
@@ -0,0 +1,221 @@
+        default_value: ''
+        values: []
+        values_method:
+        values_method_options: {}
+        options:
+          :sort_by: :value
+        label: Cluster
+        position: 2
+        validator_type:
+        validator_rule:
+        reconfigurable: false
+        dynamic: true
+        show_refresh_button: false
+        load_values_on_init: false
+        read_only:
+        auto_refresh: true
+        trigger_auto_refresh: true
+        visible: true
+        type: DialogFieldDropDownList
+        resource_action:
+          action:
+          resource_type: DialogField
+          ae_namespace: Infrastructure/VM/Transform
+          ae_class: Import
+          ae_instance: list_clusters
+          ae_message:
+          ae_attributes: {}
+      - name: storage
+        description: Target storage
+        data_type: string
+        notes:
+        notes_display:
+        display: edit
+        display_method:
+        display_method_options: {}
+        required: true
+        required_method:
+        required_method_options: {}
+        default_value: ''
+        values: []
+        values_method:
+        values_method_options: {}
+        options:
+          :sort_by: :value
+        label: Storage
+        position: 3
+        validator_type:
+        validator_rule:
+        reconfigurable:
+        dynamic: true
+        show_refresh_button: false
+        load_values_on_init: true
+        read_only:
+        auto_refresh: true
+        trigger_auto_refresh: false
+        visible: true
+        type: DialogFieldDropDownList
+        resource_action:
+          action:
+          resource_type: DialogField
+          ae_namespace: Infrastructure/VM/Transform
+          ae_class: Import
+          ae_instance: list_storages
+          ae_message:
+          ae_attributes: {}
+      - name: sparse
+        description: Use thin allocation for virtual disks
+        data_type:
+        notes:
+        notes_display:
+        display: edit
+        display_method:
+        display_method_options: {}
+        required: false
+        required_method:
+        required_method_options: {}
+        default_value: f
+        values:
+        values_method:
+        values_method_options: {}
+        options: {}
+        label: Thin provisioning
+        position: 4
+        validator_type:
+        validator_rule:
+        reconfigurable:
+        dynamic:
+        show_refresh_button:
+        load_values_on_init:
+        read_only:
+        auto_refresh:
+        trigger_auto_refresh:
+        visible: true
+        type: DialogFieldCheckBox
+        resource_action:
+          action:
+          resource_type: DialogField
+          ae_namespace:
+          ae_class:
+          ae_instance:
+          ae_message:
+          ae_attributes: {}
+  resource_actions:
+    - action:
+      ae_namespace: SYSTEM
+      ae_class: PROCESS
+      ae_instance: Request
+      ae_message:
+      ae_attributes:
```

Comment:
@matobet, Automate can be really confusing, I should have added more detail.  The solution would be to change the request value below to importvm, or some new instance name of your choosing.  You can remove the values for namespace, class, and instance here. 
You would add a new instance in the ManageIQ-content repo for /ManageIQ/System/Request class(importvm or your new name) and you would specify the relationship there to point directly to: ManageIQ/Infrastructure/VM/Transform/Import/ImportVm. Look at 
https://github.com/ManageIQ/manageiq-content/blob/master/content/automate/ManageIQ/System/Request.class/ansible_tower_job.yaml as an example.